weaver/wps_restapi/jobs/jobs.py

```diff
@@ -275,6 +340,24 @@ def get_job_results(request):
     return resp
 
 
+@sd.provider_transformer_service.get(tags=[sd.TAG_JOBS, sd.TAG_RESULTS, sd.TAG_PROVIDERS], renderer=OutputFormat.JSON,
+                                     schema=sd.ProviderTransformerEndpoint(),
+                                     response_schemas=sd.get_prov_transformer_responses)
+@sd.process_transformer_service.get(tags=[sd.TAG_JOBS, sd.TAG_RESULTS, sd.TAG_PROCESSES], renderer=OutputFormat.JSON,
+                                    schema=sd.ProcessTransformerEndpoint(),
+                                    response_schemas=sd.get_job_transformer_responses)
+@sd.job_transformer_service.get(tags=[sd.TAG_JOBS, sd.TAG_RESULTS], renderer=OutputFormat.JSON,
+                                schema=sd.JobTransformerEndpoint(), response_schemas=sd.get_job_transformer_responses)
+@log_unhandled_exceptions(logger=LOGGER, message=sd.InternalServerErrorResponseSchema.description)
+def get_job_transformer(request):
+    # type: (PyramidRequest) -> AnyResponseType
+    """
+    Retrieve the possible formats of an output job.
+    """
+    job = get_job(request)
+    return get_job_possible_output_formats(job)
```

Comment:
Should do some kind of `sd.JobTranformerSchema().deserialize(<body>)` check before returning the contents to ensure they match the format of weaver/wps_restapi/examples/job_transformer.json